VAMobile/src/store/slices/authSlice.ts

```diff
@@ -138,6 +156,10 @@ export const completeFirstTimeLogin = (): AppThunk => async (dispatch) => {
   dispatch(dispatchSetFirstLogin(false))
 }
 
+export const completeRequestNotifications = (): AppThunk => async () => {
+  await AsyncStorage.setItem(NOTIFICATION_COMPLETED_KEY, FIRST_NOTIFICATION_STORAGE_VAL)
+}
+
```

Comment:
This probably doesn't need to be a function, since it's just setting a value in Asnyc Storage.

Reply:
Removed. I needed to drop the await in order to include it in the screen itself, but that shouldn't be a problem I don't think.

Reply:
Yeah that shouldn't cause any issues since there's not anything that needs to wait for async storage to finish setting the value